built-in-references/Kubernetes/container-probes-required/template.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sazurecontainerprobesrequired
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sAzureContainerProbesRequired
+        listKind: K8sAzureContainerProbesRequiredList
```

Comment:
Only kind should be defined

Reply:
I removed the list kind. Please check if it ok now.